protocol/x/bridge/keeper/complete_bridge.go

```diff
@@ -35,14 +35,18 @@ func (k Keeper) CompleteBridge(
 		return err
 	}
 
-	// Send coin from bridge module account to specified account.
-	if err = k.bankKeeper.SendCoinsFromModuleToAccount(
-		ctx,
-		types.ModuleName,
-		bridgeAccAddress,
-		sdk.Coins{bridge.Coin},
-	); err != nil {
-		return err
+	// If coin amount is positive, send coin from bridge module account to
+	// specified account.
+	if bridge.Coin.Amount.IsPositive() {
```

Comment:
I see we have taken the approach to still schedule delayed messages for `0` transfers. 
Can you document somewhere the rationale behind tradeoffs of different approaches, and why we are allowing zero-transfer delay messages to be added to state?

Reply:
Documented rationale in PR description!